OpenRA.Game/TextNotificationsManager.cs

```diff
@@ -20,6 +21,9 @@ public static class TextNotificationsManager
 		static readonly string SystemMessageLabel;
 
 		public static long ChatDisabledUntil { get; internal set; }
+		static readonly List<TextNotification> NotificationsCache = new List<TextNotification>();
+
+		public static IReadOnlyList<TextNotification> Notifications => NotificationsCache;
```

Comment:
Just a small nit, IMO there shouldn't be a space between `NotificationsCache` and `Notifications`